src/nvim/generators/gen_options.lua

```diff
@@ -163,6 +160,12 @@ local dump_option = function(i, o)
     defines['PV_' .. varname:sub(3):upper()] = pv_name
     w('    .indir=' .. pv_name)
   end
+  if o.enable_if then
+    w('#else')
+    w('    .var=NULL')
```

Comment:
Vim does this for `shellslash`. @skippi Any issues from ASAN if you apply this patch to your PR?

Reply:
This LGTM. No ASAN issues for my patch.